src/styles/Rockets.css

```diff
@@ -0,0 +1,19 @@
+.rockets .card {
+  display: flex;
+  gap: 15px;
+  margin-top: 20px;
+}
```

Comment:
I suggest that you give it some padding, so it will match the design in the requirements

Your Padding:
![padding2](https://user-images.githubusercontent.com/95412943/208611973-611ab878-6df6-4a36-a779-e5305612c488.PNG)

The reqs Padding:
![reqPadding](https://user-images.githubusercontent.com/95412943/208612054-2ba00404-5804-4a7e-a40b-5a681399d7b6.PNG)

Cheers 👍

Reply:
@Abdessittir 

Thank you for your review and your suggestion I'll fix it of course 🙂